classes/controller/FrontController.php

```diff
@@ -454,12 +454,6 @@ public function init()
             }
         }
 
-        $languages = Language::getLanguages(true, $this->context->shop->id);
-        $meta_language = array();
-        foreach ($languages as $lang) {
-            $meta_language[] = $lang['iso_code'];
-        }
```

Comment:
Not really related to the PR, but these lines were executed for nothing (variable not used).